### DISCUSSION TITLE
Do you accept contributions to documentation translations?

### DISCUSSION THREAD

**Discussion by Winson-Huang** [neovim/neovim#27107] [Q&A] [answered]
Do you really accept contributions to neovim documentation translations, just as described in [helphelp](https://neovim.io/doc/user/helphelp.html#help-translated)? I don't find related issues so I come here to ask.

**Comment by clason** [comment]


**Reply by clason** [reply, marked as answer]
Oh, sorry, _documentation_. Then, no, not right now. The documentation is not stable enough yet, and partly autogenerated.

(This help file is included from Vim.)

**Reply by Winson-Huang** [reply]
Alright, then is there any guidance on improving existing documentations? I really think good docs are important for users, especially for beginners.

**Reply by clason** [reply]
If there are parts you think are lacking, open an issue explaining what exactly you are missing. (Most of our documentation is a technical reference manual, so conciseness is important. We don't have much of a "user manual" yet, apart from `usr_*.txt` inherited from Vim -- which doesn't cover Neovim specifics -- and `luaguide.txt`.)
